data/abilities.js

```diff
@@ -811,10 +812,13 @@ exports.BattleAbilities = {
 					showMsg = true;
 				}
 			}
-			if (showMsg && !effect.secondaries) this.add("-fail", target, "unboost", "[from] ability: Flower Veil", "[of] " + target);
+			if (showMsg && !effect.secondaries) this.add('-activate', this.effectData.target, 'ability: Flower Veil', '[of] ' + target);
```

Comment:
If this change is correct, then shouldn't the other stat drop blocking abilities be changed too?